### PR TITLE
Create immersive article templates and style header

### PR DESCRIPTION
**Pull request by sammorrisdesign-zz**
# ARE YOU READY TO BE IMMERSED!?!¡?

This takes advantage of the new `immersive` switch added to composer for all articles – it previously was only available for `interactives`. This new immersive article is designed for our longer reads and [long reads](http://www.theguardian.com/news/series/the-long-read) to have the nicer presentation we want and they deserve, but also to maintain some consistency. This is just the initial PR, there will be more to come, mainly styling.

I've created a new a new article template as we'll be deviating from the basic one quite a bit and it avoids death by a hundred if statements. The rest should be fairly self-explanatory.

Here's the screenshot bit...

Mobile:
screen shot 2015-10-08 at 16 36 11

Desktop:
screen shot 2015-10-08 at 16 40 49

### CODE REVIEW

File: article/app/views/fragments/articleBodyImmersive.scala.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <div class="l-side-margins">
+        <article id="article" data-test-id="article-root"
+            class="content content--article content--immersive tonal tonal--@articleToneClass(article) section-@article.sectionName.trim.toLowerCase.replaceAll("""[\s-]+""", "-")
+            @if(article.isAdvertisementFeature){content--advertisement-feature}
+            @if(article.isFeature && article.hasShowcaseMainElement){has-feature-showcase-element}"
+            itemscope itemtype="@article.schemaType" role="main">
+            <meta itemprop="mainEntityOfPage" content="@LinkTo(article.url)">
+            <div itemprop="publisher" itemscope itemtype="https://schema.org/Organization">
+                <meta itemprop="name" content="The Guardian">
+                @if(amp) {
```
Comment (janua) [inline article/app/views/fragments/articleBodyImmersive.scala.html]: Under what circumstances is an **Immersive!?!¡?** article also an `AMP` article?

**Comment by janua** [comment]
Apart from the amp stuff that is always false, looks good T_T :+1:

File: article/app/views/fragments/articleBodyImmersive.scala.html (hunk 1 above)
Comment (sammorrisdesign) [inline article/app/views/fragments/articleBodyImmersive.scala.html]: This was copied from the main `articleBody.scala.html` template. I wonder if this is just a remnant from that @stephanfowler / @NataliaLKB ?

If it is |'ll remove it from here and the main Body template.

Comment (janua) [inline article/app/views/fragments/articleBodyImmersive.scala.html]: I think you can probably remove it, as it's based off a `boolean` passed into the template that you never seem to set.
There's another one further down.

Comment (NataliaLKB) [inline article/app/views/fragments/articleBodyImmersive.scala.html]: Hello @sammorrisdesign @janua. AMP isn't entirely different templates anymore, as there is not a lot that we are changing regarding the structure of an article. 

Do keep the AMP if statement here :smile:

Comment (janua) [inline article/app/views/fragments/articleBodyImmersive.scala.html]: @NataliaLKB This is an entirely different template though. If is it an `AMP` article you will never come in here as the code is branching before this in `ArticleController`.

Comment (NataliaLKB) [inline article/app/views/fragments/articleBodyImmersive.scala.html]: @janua Oh yes I didn't see that. so the answer is to remove for this template, but keep it in the main one. 

I would imagine we would want immersives in AMP as well though ... @stephanfowler @johnduffell ?

Comment (stephanfowler) [inline article/app/views/fragments/articleBodyImmersive.scala.html]: Yes we want the content - but (unless/until we replicate their styling etc.) can they fall back to being rendered as if they were a regular article?

**Comment by sammorrisdesign** [comment]
Just removed the amp stuff then. Let's do some immersing

**Comment by johnduffell** [comment]
never mind special articles, let's make the whole site like this! :stuck_out_tongue_winking_eye:

**Pull request merged**